### ISSUE TITLE
[Feature] Standalone welcome / status page for CODE server

### ISSUE THREAD

**Issue by madmath03** [CollaboraOnline/richdocumentscode#48]
CODE provides a welcome page at `/dist/welcome/welcome.html` which is designed to be displayed **inside the editor**, but there is no standalone welcome / status page to see if the server is correctly running.

My idea would be to:

1. create a standalone welcome page at `/dist/welcome/index.html`
2. integration the already existing `welcome.html` (iframe ?) into the standalone welcome page
3. add a simple JS script to ping the server and get its status (kind of like in CollaboraOnline/Docker-CODE#100)
4. Display the server's status before or after the welcome message

For comparison, ONLYOFFICE does this with a simple page like this: 
![image](https://user-images.githubusercontent.com/6967675/88494643-444bbb00-cfb7-11ea-94f7-5ede9fdb6430.png)

**Comment by raffisweb** [comment]
Good idea! Don't know if a ping is enough. Had some https issues that can't be recognized with a ping. So some more tests are welcome. ;-)
